src/styles/card-block.scss

```diff
@@ -0,0 +1,97 @@
+$width: 200px;
+$main-text-color: #060b35;
+$link-buy-color: #00acdc;
+
+.card {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 12px;
```

Comment:
Consider using a variable for the font-family to maintain consistency across the project.